galaxyproject/galaxy
# [WIP] Allow router to reactively control the masthead.

The goal here is to replace the hacks in the workflow centric UI where we want a completely different masthead depending on the router we're rendering. This just sets up the plumbing though and uses restoring the highlight of the current masthead tab selected from older Galaxies with the new version of the masthead.

I talked to @mvdbeek and he thought dropping the highlighted tab was a regression. If instead we want it to no be highlighted, I can drop that but I would still like to get the core refactoring in for the downstream work ideally.

## How to test the changes?
(Select all options that apply)
  1. Click around the UI and see the current tab is selected again. Here is an older Galaxy (http://huttenhower.sph.harvard.edu/galaxy/) that used to do this as well.

## License
